examples/r2i/tflite/Makefile.am

```diff
@@ -27,8 +27,9 @@ AM_CPPFLAGS =                      \
         $(RR_CPPFLAGS)             \
         $(CODE_COVERAGE_CPPFLAGS)
 
-AM_LDFLAGS =                       \
+LDADD =                            \
         $(RR_LIBS)                 \
+        $(TFLITE_LIBS)             \
```

Comment:
This should no longer be necessary. Libtool should handle the inter-library dependencies.